### ISSUE TITLE
bug: 🐝 File not found `v4/lib/main.js`

### ISSUE THREAD

**Issue by sebnitu** [JamesIves/github-pages-deploy-action#1697]
### Describe the bug

Currently getting this error when the deploy step is reached:

```
Run JamesIves/github-pages-deploy-action@v4
Error: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'
```

### Reproduction Steps

<img width="858" alt="image" src="https://github.com/user-attachments/assets/613bb018-fe14-4114-8213-6211563a9959">

### Logs

```shell
##[debug]Evaluating condition for step: 'Deploy'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Deploy
##[debug]Loading inputs
##[debug]Evaluating: github.token
##[debug]Evaluating Index:
##[debug]..Evaluating github:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'token'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Loading env
Run JamesIves/github-pages-deploy-action@v4
Error: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'
##[debug]System.IO.FileNotFoundException: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'
##[debug]File name: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'
##[debug]   at GitHub.Runner.Sdk.ArgUtil.File(String fileName, String name)
##[debug]   at GitHub.Runner.Worker.Handlers.NodeScriptActionHandler.RunAsync(ActionRunStage stage)
##[debug]   at GitHub.Runner.Worker.ActionRunner.RunAsync()
##[debug]   at GitHub.Runner.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken)
##[debug]Finishing: Deploy
```

### Workflow

```yml
name: Deploy

on:
  push:
    branches: [main]

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
        with:
          node-version: 16.13.0

      - name: Setup
        run: npm ci
  
      - name: Build
        run: npm run build

      - name: Deploy
        uses: JamesIves/github-pages-deploy-action@v4
        with:
          branch: gh-pages
          folder: dist
```
```


### Additional Comments

_No response_

**Comment by vferat** [comment]
same issue on my side

**Comment by ben-d-marks** [comment]
Same

**Comment by Ncookiez** [comment]
Until this issue is fixed:
Change `uses: JamesIves/github-pages-deploy-action@v4` to `uses: JamesIves/github-pages-deploy-action@v4.6.4`
Confirmed working on my end.

**Comment by pingu2k4** [comment]
Same issue here. Using similar to the solution above, but pinning to the commit hash to revert to previous version:

```
uses: JamesIves/github-pages-deploy-action@920cbb300dcd3f0568dbc42700c61e2fd9e6139c # v4
```

**Comment by mahima98** [comment]
Same issue on my side and tomorrow am doing a presentation demo on "Exploring Nuxt Studio" 🥹

**Comment by ml-james** [comment]
Same issue here.

**Comment by Anchorsify** [comment]
Same issue as well.

**Comment by MrSunshyne** [comment]
same issue

**Comment by MrSunshyne** [comment]
For now i've fixed this by changing the version to 4.6.4 manually in the workflow file instead of v4.

![image](https://github.com/user-attachments/assets/258d6388-f779-4bbf-b68e-9fac87822ce5)

Here [the reproduction repo](https://github.com/MrSunshyne/starter-test-frontendmu/actions)

**Comment by splincode** [comment]
The same error:
https://github.com/taiga-family/ng-morph/actions/runs/11075647463/job/30777054778

**Comment by d33bs** [comment]
Also seeing this issue occur. Thank you for any fixes and for the work in general on this project!

**Comment by remarkablemark** [comment]
I'm also getting this error: https://github.com/remarkablegames/haunted-heir/actions/runs/11079361676/job/30788302147

**Comment by yongzx** [comment]
Still seeing the same issue
<img width="1503" alt="image" src="https://github.com/user-attachments/assets/433aa8e6-7949-4ec5-be81-8425338e45e6">

**Comment by fscdc** [comment]
same issue! it works out!

**Comment by peterschmidt85** [comment]
Same issue. Any update on this?

**Comment by fscdc** [comment]
> Same issue. Any update on this?

v4 -> v4.6.4

**Comment by 246859** [comment]
same issue

**Comment by JamesIves** [comment]
This issue has now been resolved as of [v4.6.7](https://github.com/JamesIves/github-pages-deploy-action/releases/tag/v4.6.7), apologies about that. I'm going to make some adjustments to the integration tests so this doesn't go missed during the release cycle. The `v4` major tag has also been updated so any existing workflows should now work as expected.
